equinix/data_source_fabric_cloud_router.go

```diff
@@ -38,3 +140,15 @@ func dataSourceFabricCloudRouterRead(ctx context.Context, d *schema.ResourceData
 	d.SetId(uuid)
 	return resourceFabricCloudRouterRead(ctx, d, meta)
 }
+
+func dataSourceFabricGetCloudRouters() *schema.Resource {
```

Comment:
I understand that there may be some shared code in play here, but it would be preferable to put the new data source in its own file.

Reply:
Solid feedback. Updated.